homeassistant/components/vera/sensor.py

```diff
@@ -56,7 +57,7 @@ def __init__(
         VeraDevice.__init__(self, vera_device, controller_data)
         self.entity_id = ENTITY_ID_FORMAT.format(self.vera_id)
 
-    @property
+    @cached_property
```

Comment:
I think this can be a separate PR as well to move it to the constructor

Reply:
#100269